tbx/project_styleguide/templates/patterns/pages/errors/404.html

```diff
@@ -1,11 +1,24 @@
 {% extends "patterns/base_page.html" %}
 
 {% block title %}Page not found{% endblock %}
+
+{% block body_class %}template-404-page mode-dark{% endblock %}
```

Comment:
This has alerted me to the fact that the `body_class` block is badly named as it is on the html. It would also be best just to set the mode and theme in `base.html`.

So, I would suggest changing the base html `html` tag to the following:

```
<html
    class="
   {% if page.theme_class %}{{ page.theme_class }}{% endif %} mode-dark
   {% block html_class %}
        template-{{ page.get_verbose_name|slugify }}
    {% endblock %}"
    lang="en-GB"
>
```

There is then just one other template to update to use `html_class`

Reply:
I can't figure out why at the moment but the CSS for the 404 page is not loading correctly for me

<img width="1068" alt="Screenshot 2024-03-22 at 14 29 48" src="https://github.com/torchbox/torchbox.com/assets/771869/5c268639-3a64-4865-9244-f22a9affda3f">

Reply:
Turns out my css wasn't compiling because I had `debug=False` in my local settings for a different reason.

Reply:
_So that's why_ I kept having to run `dj collectstatic` manually! I had the same issue. 😅

Reply:
Renamed `body_class` to `html_class` in [93d82be](https://github.com/torchbox/torchbox.com/pull/182/commits/93d82be03e40e763d941f5a79af5b6dc19715eca).